backend/src/Controllers/getElectionsController.ts

```diff
@@ -0,0 +1,22 @@
+import ServiceLocator from '../ServiceLocator';
+import Logger from '../Services/Logging/Logger';
+import { responseErr } from '../Util';
+import { BadRequest } from "@curveball/http-errors";
+
+var ElectionsModel = ServiceLocator.electionsDb();
+
+const getElections = async (req: any, res: any, next: any) => {
+    Logger.info(req, `getElections`);
+    var filter = (req.query.filter == undefined) ? "" : req.query.filter;
+    const Elections = await ElectionsModel.getElections(filter, req);
+    if (!Elections) {
+        var msg = "Election does not exist";
+        Logger.info(req, msg);
+        throw new BadRequest(msg);
+    }
+    res.json(Elections);
```

Comment:
seems we normally return something like 
res.json( { elections : Elections })

so would be good to be consistent (unless specific need to deviate, if so maybe add comment)

Reply:
Updated in recent commit